cmd/gitquery/query_base.go

```diff
@@ -17,7 +17,7 @@ import (
 type cmdQueryBase struct {
 	cmd
 
-	Path string `short:"p" long:"path" description:"Path where the git repository is located"`
+	Path []string `short:"p" long:"path" description:"Path where the git repository is located, can be used several times"`
```

Comment:
Maybe this should be just a string instead of a slice and internally check the repositories inside that path. The description should be something like: `path where the git repositories are located`

Reply:
Should we also support bucketing?

Reply:
This first version does not support bucketing.